### PR TITLE
Adding support to mod_disk_cache for >= 2.4

### PR DESCRIPTION
**Pull request by JamieCressey**
Expanding on the initial PR by @dteirney (#1037) - Adding support to mod_disk_cache for >= 2.4

### CODE REVIEW

**Comment by JamieCressey** [comment]
Correct, good spot. Updated and tests passed successfully again.

**Comment by hunner** [comment]
Looks like there are merge conflicts. Could you rebase please?

**Comment by JamieCressey** [comment]
@hunner rebased and tests passed.

File: templates/mod/disk_cache.conf.erb [hunk 1]
```diff
@@ -1,8 +1,4 @@
-<IfModule mod_proxy.c>
-  <IfModule mod_disk_cache.c>
-     CacheEnable disk /
-     CacheRoot "<%= @cache_root %>"
-     CacheDirLevels 2
-     CacheDirLength 1
-  </IfModule>
-</IfModule>
```
Comment (igalic) [inline templates/mod/disk_cache.conf.erb]: ♥

**Comment by igalic** [comment]
@JamieCressey thanks for the rebase.
would you mind doing a [squash](http://www.git-scm.com/book/en/Git-Tools-Rewriting-History#Squashing-Commits) as well? (i'm not very happy with a commit titled 'Merging README conflict', and i think this stuff is closed-up enough to only deserve one commit)

**Comment by JamieCressey** [comment]
@igalic Voila! Let me know if you need anything else to get this merged. Thanks.

**Comment by igalic** [comment]
thanks @JamieCressey!

**Pull request merged**